src/assets/audio/audio_map_points.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
Was this handwritten or parsed from elsewhere? I couldn't find this in the original boundary or asset directory

Reply:
Brad replied that he hand wrote it for the demo